ls1intum/Artemis
# `Development`: Fix syntax error preventing syntax highlighting in server guidelines

### Motivation, Context and Description
When compiling the documentation, some code blocks get marked as parsing errors. This due to a syntax error in the example code which is fixed now.

Open Question: Looking at the code the call to `request.postWithResponseBodyFile` also requires some export options. This is not relevant for the example, but also makes it outdated. Should I add these options into the documentation as well?

### Steps for Testing
1. Go to the documentation built from this pull-request (readthedocs GitHub action)
2. Navigate to the server code guidelines
3. Check that the last two java code examples (for testing) are highlighted correctly now

### Review Progress

#### Code Review
